tests/__init__.py

```diff
@@ -0,0 +1,52 @@
+import io
+import os
+import sys
+import argparse
+import subprocess
+from contextlib import redirect_stdout
+from tempfile import NamedTemporaryFile
+from typing import List, Callable
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from tests import config
```

Comment:
Unused import?

Reply:
Nope, it configures the test environment.

Reply:
I'll add a comment.